packages/libs/client/src/integration-tests/test-data/accounts.ts

```diff
@@ -0,0 +1,14 @@
+import { IAccount } from '../support/interfaces';
+
+export const sourceAccount: IAccount = {
+  account: 'k:5a2afbc4564b76b2c27ce5a644cab643c43663835ea0be22433b209d3351f937',
+  publicKey: '5a2afbc4564b76b2c27ce5a644cab643c43663835ea0be22433b209d3351f937',
+  chainId: '0',
+  guard: '5a2afbc4564b76b2c27ce5a644cab643c43663835ea0be22433b209d3351f937',
+};
+export const targetAccount: IAccount = {
```

Comment:
Why is this one the same as sourceAccount?

Reply:
I opted for the usecase that an individual wants to transfer funds to his own account on another chain. I can easily update if it's more realistic to be a different account?